### PR TITLE
Allow plugins to be run in separate process

### PR DESCRIPTION
**Pull request by pylanglois**
Hi yapsy team,

Thanks for yapsy, it is a great software, very well designed, light and easy to use!

I would like to use yapsy for my project but I need each plugin to run in separate process. So I created a set of classes that achieve this goal. I tried to code it as minimalistic as possible to keep the maintenance simple.

What do you think of this feature?

Thanks!

Pierre-Yves Langlois

### CODE REVIEW

File: package/yapsy/__init__.py [hunk 1]
```diff
@@ -53,7 +53,7 @@
 
 """
 
-__version__="1.10.423"
+__version__="1.10.423.py_9"
```
Comment (tibonihoo) [inline package/yapsy/__init__.py]: Why this exactly ? I won t merge this precise bit of course.

**Comment by tibonihoo** [comment]
Wow this looks very interesting indeed ! I'll merge that to the original repo on yaps in a few days.

File: package/yapsy/__init__.py (hunk 1 above)
Comment (pylanglois) [inline package/yapsy/__init__.py]: I did this to test the integration with my system but I forgot to revert the change. Sorry!

**Comment by pylanglois** [comment]
Thanks!

**Comment by tibonihoo** [comment]
Hello again,

Now that this nice feature has been merged, I'm planning a release of yapsy. Traditionally, I like asking contributors about their projects where they use yapsy so that I can put a link in the "showcase" section of the doc.

Would it be possible for me to give a link to your project ?

Obviously, there is absolutely no obligation for you and no offense will be taken if you don't want your project to appear there.

**Comment by pylanglois** [comment]
Hello,

Thank you for merging my code. I'm happy to see this feature is
appreciated!

My project is still in early development but the core is going well. For
now the code is not public because it is too ugly/prototyped to show the
world :)

I plan to release it on github later this spring and I will send you the
link when it is ready.

Thank again,

Pierre-Yves
